docs/sql-ref-identifier-clause.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: global
+title: Identifier clause
+displayTitle: IDENTIFIER clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+Convert a STRING literal into an identifier or name. The IDENTIFIER clause can be used in many places instead of, or in combination with identifiers where identifiers or names are used.
```

Comment:
Shall we be more clear here? I think we can say that this clause can be used anywhere that an expression can be used, and will be converted into an unresolved column. In addition, it can also be used where a table identifier is used, such as FROM tbl, CREATE TABLE tbl, and will be converted into an unresolved relation.

Reply:
It's a game of whack-a-mole. I did not cover all places. I did not cover all places, And I'm unsure how feasible it is to cover all (and keep them covered).